src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Comparison/UniquePatternService.cs

```diff
@@ -0,0 +1,75 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Drawing;
```

Comment:
should we remove this namespace?  I don't think we should have any UI related code in this package.